README.md

```diff
@@ -20,9 +20,53 @@ This chart deploys a fully functioning instance of the [Fn](https://github.com/f
 
 
 ```bash
-helm init
+helm init --upgrade
 ```
 
+## Preparing chart values
+
+### Minimum configuration
+
+In order to get a working deployment please pay attention to what you have in your chart values.
+[Here](fn/values.yaml) you may find bare minimum chart values configuration necessary to deploy a working Fn cluster.
+
+### Exposing Fn services
+
+#### Ingress controller
+
+If you are installing Fn behind an ingress controller you'd need to have only 1 DNS sub-domain that will stand for an ingress controller IP resolve.
```

Comment:
If you are installing Fn behind an ingress controller, you'll need to have a single DNS subdomain that will act as your ingress controllers IP resolution. (i think this says what you're trying to say?)